modules/nodes-scheduler-taints-tolerations-about.adoc

```diff
@@ -216,9 +244,23 @@ These tolerations ensure that the default pod behavior is to remain bound for fi
 
 You can configure these tolerations as needed. For example, if you have an application with a lot of local state, you might want to keep the pods bound to node for a longer time in the event of network partition, allowing for the partition to recover and avoiding pod eviction.
```

Comment:
What kind of event is _network partition_? Is it a network-based PV that becomes unavailable or something?

Reply:
"A network partition refers to a network split between nodes due to the failure of network devices" Also stolen from elsewhere. 
The idea is, if the control plane loses communication with a node, it add a `node.kubernetes.io/unreachable` taint. If the pods on that node do not tolerate that taint, they are removed. If you want the pods to stay for a period, in the hope that the node comes back, use `tolerationSeconds`. If the pods are important, you might want them to keep the pod running until you know the node is gone.